test/dynamo/test_misc.py

```diff
@@ -4005,6 +4005,7 @@ def fn(x, y):
             root_tx=None,
             export=False,
             export_constraints=None,
+            dynamic_plan=None,
```

Comment:
out of date

Reply:
Everywhere you say `dynamic_plan` it needs to be updated

Reply:
yea, this has not been revved yet